src/db/entry.rs

```diff
@@ -41,6 +41,63 @@ impl Entry {
             ..Default::default()
         }
     }
+
+    pub(crate) fn merge(&mut self, other: &Entry) {
+        if self.times.get_last_modification().unwrap()
```

Comment:
This is so verbose just to get a comparable time. Can we introduce a function in times to compare one times to another and you pass in an enum to choose which field (creation, modification, access)?

Reply:
We end up having to do multiple comparisons with the last modification times, so I now extract them early in the merge function [here](https://github.com/sseemayer/keepass-rs/pull/155/files#diff-83551f8ef922bfe6845bfd0c8ca2fab09b6be786474426c39f6c1ea3c7e37402R48). I'll keep youfr proposition in mind in case we have to compare the last modification times in other functions in the future.